test/remote/gateways/remote_payeezy_test.rb

```diff
@@ -64,7 +92,7 @@ def test_failed_authorize
     @amount = 501300
     assert auth = @gateway.authorize(@amount, @credit_card, @options)
     assert_failure auth
-    assert auth.authorization
+    assert_equal nil, auth.authorization
```

Comment:
This reads like `purchase` used to return an authorization structure, but will now be `nil`. Seems like a compatibility issue?

Reply:
@dtykocki Restructured the way the authorization is built to accommodate the failed store response and returned these tests to the way they were.